components/measurement/CommonDetails.js

```diff
@@ -35,9 +35,9 @@ const StyledReactJsonContainer = styled.div`
   }
 `
 
-const JsonViewer = ({ src }) => (
+const JsonViewer = ({ src, collapse }) => (
   <StyledReactJsonContainer>
-    <ReactJson collapsed={1} src={src} />
+    <ReactJson collapsed={collapse} src={src} />
```

Comment:
While you are at this, can you also add these props to the json viewer component? ` name={null} indentWidth={2}`

Reply:
Done